products/monitoring/api.yaml

```diff
@@ -1094,6 +1096,127 @@ objects:
                   A duration string, e.g. 10s.
                   Good service is defined to be the count of requests made to
                   this service that return in no more than threshold.
+        - !ruby/object:Api::Type::NestedObject
+          name: requestBasedSli
+          api_name: 'requestBased'
+          exactly_one_of:
```

Comment:
Totally optional, but I think it would be nice to put the exactly one of info in the docs for each field just to make it super obvious for people what they do and don't have to set.

Reply:
👍